LEGO1/legoentity.cpp

```diff
@@ -0,0 +1,107 @@
+#include "legoentity.h"
+
+// OFFSET: LEGO1 0x100105f0 STUB
+LegoEntity::LegoEntity()
```

Comment:
Incorrect offset.
`LegoEntity` is created `inLegoObjectFactory::Create` at around `1000a3f3`.

Reply:
(It's inlined)